modules/cnv-creating-vddk-image.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-importing-vmware-vm.adoc
+
+[id="cnv-creating-vddk-image_{context}"]
+= Creating the Virtual Disk Development Kit image
+
+You can download the VMware Virtual Disk Development Kit (VDDK) SDK, convert it into an image, and push it to a container repository.
+
+.Prerequisites
+
+* *My VMware* credentials to download the VDDK SDK
+* Private container repository. Storing the VDDK SDK in a public repository may violate the terms of the VMware license.
+
+.Procedure
+
+. Create a temporary directory for VDDK:
++
+----
+$ mkdir /tmp/my_vddk && cd /tmp/my_vddk
+----
+
+. Download the VDDK SDK:
+.. Navigate to link:https://www.vmware.com/support/pubs/[VMware Documentation].
```

Comment:
s/Navigate to/In a web browser, open the/

Reply:
"Open" a web site? Are you sure? I did a global search and this phrasing is not used in Openshift docs.

Reply:
> @apinnick which version of OpenShift is this targeted for? Thanks.

CNV 2.1.